config/nrfconnect/chip-module/Kconfig.defaults

```
@@ -199,11 +199,15 @@ config CHIP_QSPI_NOR
     default y if BOARD_NRF5340DK_NRF5340_CPUAPP || BOARD_NRF52840DK_NRF52840
 
 # nRF7002DK uses SPI NOR external flash
+
+if BOARD_NRF7002DK_NRF5340_CPUAPP || BOARD_NRF7002DK_NRF7001_NRF5340_CPUAPP
+
 config CHIP_SPI_NOR
     default y if BOARD_NRF7002DK_NRF5340_CPUAPP
```

Did you mean the following?
```suggestion
    default y if BOARD_NRF7002DK_NRF5340_CPUAPP || BOARD_NRF7002DK_NRF7001_NRF5340_CPUAPP
```
We support also other nRF53 boards so `BOOT_IMAGE_ACCESS_HOOKS` should be set for them as well.